back/db/migrate/20230815182301_add_cosponsors_initiative_to_notifications.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+class AddCosponsorsInitiativeToNotifications < ActiveRecord::Migration[7.0]
+  def change
+    add_column :notifications, :cosponsors_initiative_id, :uuid, null: true
```

Comment:
Can we use `post_id` (`initiative_id`) instead of `cosponsors_initiative_id` the same as for other initiative-related notifications? It seems the number of columns in this table went out of control.

Reply:
Not sure I understand. The activity item is a `cosponsors_initiative` item.

Reply:
Can the activity item be an `initiative`? It would be great if we didn't have this new column and grouped all these new notifications/emails around an initiative.

Reply:
Discussed on a call, and while it is possible (we would need to do something like pass the data of a `cosponsors_initiative` in a `'cosponsors_initiative_created'` activity with an `initiative` as the activity item), we agreed that we will stick with the conventional approach (where the activity item is the item being created/changed) for now, as the deadline for this work is approaching.